learn/core_concepts/documents.md

```diff
@@ -48,7 +48,7 @@ To learn more, refer to the [primary key explanation](/learn/core_concepts/prima
 
 By default, Meilisearch limits the size of all payloads—and therefore document uploads—to 100MB. You can [change the payload size limit](/learn/configuration/instance_options.md#payload-limit-size) at runtime using the `http-payload-size-limit` option.
 
-Meilisearch uses a lot of RAM when indexing documents. Be aware of your [RAM availability](/resources/faq.md#what-are-the-recommended-requirements-for-hosting-a-meilisearch-instance) as you increase your batch size as this could cause Meilisearch to crash.
+Meilisearch uses a lot of RAM when indexing documents. Be aware of your [RAM availability](/faq.md#what-are-the-recommended-requirements-for-hosting-a-meilisearch-instance) as you increase your batch size as this could cause Meilisearch to crash.
```

Comment:
```suggestion
Meilisearch uses a lot of RAM when indexing documents. Be aware of your [RAM availability](https://docs.meilisearch.com/faq.html#what-are-the-recommended-requirements-for-hosting-a-meilisearch-instance) as you increase your batch size as this could cause Meilisearch to crash.
```